src/gui/transferlistmodel.cpp

```diff
@@ -55,6 +55,73 @@ static QIcon getErrorIcon();
 
 static bool isDarkTheme();
 
+TransferListTextColors::TransferListTextColors()
+{
+    m_unknownTextColor = getColorByState(BitTorrent::TorrentState::Unknown);
```

Comment:
getColorByState is redundant now so you should assign values directly. Also please use field initializers instead of constructor body.

Reply:
multiple states have the same color, that's why I did it this way.

Will convert to constructor body.

Reply:
Initialization in the constructor line is done.

Should I remove the `getColorByState`? 
IDK I prefer it this way since there is same color for multiple states and two states. but if you insist I will remove the `getColorByState`.